cli/ballerina-cli/src/main/resources/create_cmd_templates/service/service.bal

```diff
@@ -1,21 +1,19 @@
 import ballerina/http;
-import ballerina/io;
 
 # A service representing a network-accessible API
-# bound to absolute path `/hello` and port `9090`.
-service /hello on new http:Listener(9090) {
+# bound to port `9090`.
+service hello on new http:Listener(9090) {
 
     # A resource respresenting an invokable API method
     # accessible at `/hello/sayHello`.
-    #
-    # + caller - the client invoking this resource
-    # + request - the inbound request
-    resource function get sayHello(http:Caller caller, http:Request request) {
-
+    # add return
+    resource function sayHello(string name) returns string {
```

Comment:
We can make the parameter to have a default value here and implement the `if` condition in L13 accordingly